demo/basic.html

```diff
@@ -0,0 +1,149 @@
+<!doctype html>
```

Comment:
As far as I know, all demos on one page and iron-demo-helpers / demo-snippet is the best way to go right now and most compatible with webcomponents.org.

See https://www.webcomponents.org/element/PolymerElements/paper-button/demo/demo/index.html for example